ai-data/llm-inference/quickstart.mdx

```diff
@@ -10,16 +10,21 @@ categories:
   - ia
 ---
 
-Scaleway LLM Inference service, a scalable and secure inference engine for machine learning models.
+Scaleway LLM Inference service, the 1st European LLM Inference platform on the market, is a scalable and secure inference engine for large language models.
 
-Scaleway LLM Inference is a fully managed service that allows you to run machine learning models in a production environment.
-With Scaleway LLM Inference, you can easily deploy, manage, and scale your inference models without worrying about the underlying infrastructure.
+Scaleway LLM Inference is a fully managed service that allows you to serve generative AI models in a production environment.
+With Scaleway LLM Inference, you can easily deploy, manage, and scale LLMs without worrying about the underlying infrastructure.
 
-Here are some of the key features of Scaleway Inference:
+Here are some of the key features of Scaleway Inference service:
 
-* **Easy deployment**: Deploy your inference models with just a few clicks. Scaleway Inference provides a simple and intuitive interface for uploading and deploying models.
-* **Auto-scaling**: Scaleway LLM Inference automatically scales your inference instances based on the demand, ensuring that your models are always available and responsive.
-* **Security**: Scaleway provides a secure environment for running your inference models. Our platform is built on top of a secure architecture, and we use state-of-the-art security.
+* **Easy deployment**: Deploy state-of-the-art open weights LLMs with just a few clicks. Scaleway Inference provides a simple and intuitive interface for generating dedicated endpoints.
+* **Security**: Scaleway provides a secure environment for running your models. Our platform is built on top of a secure architecture, and we use state-of-the-art security.
+* **Complete data privacy**: No storage nor third-party access to your data (prompt & responses), ensuring it remains exclusively yours and within Europe.
+* **Auto-scaling (soon!)**: Scaleway LLM Inference automatically scales your instances based on the demand, ensuring that your models are always available and responsive.
+
+<Message type="important">
+  This service is in private beta. [Specific Terms and Conditions](https://www.scaleway.com/en/contracts/) are applicable.
```

Comment:
```suggestion
  This service is in private beta. [Terms and conditions](https://www.scaleway.com/en/contracts/) apply.
```

Reply:
It was meant to say we have specific terms as opposed to the general terms of services. In fact, the beta specific terms take precedence over our general terms of service.

Reply:
OK I've added back the word specific